backend/server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/ServerApi.java

```diff
@@ -69,7 +77,7 @@ public OrganizationApi organization() {
   }
 
   public IssueApi issue() {
-    return new IssueApi(helper);
+    return new IssueApi(helper, wrongTokenCallbackWithConnectionIdParam);
```

Comment:
Why is the IssueApi the only one that accepts this parameter, and not other Api classes?